task_line/src/main.cpp

```diff
@@ -20,11 +20,11 @@ void lineCenterDataCallback(const std_msgs::Float64MultiArrayPtr &_msg) {
     if (coord_count == 0) {
         coord_count++;
         y_coord.setReference(0);
-        x_coord.setReference(0);
+        // x_coord.setReference(0);
```

Comment:
Doesn't really matter whether it is set or not, but it may prevent forward drag along the line.